.env-example

```diff
@@ -4,3 +4,6 @@ MAILTRAP_PASSWORD=InboxPassword
 # Camo proxies insecure images in students' status updates
 CAMO_HOST=https://rgsoc-teams-camo.herokuapp.com
 CAMO_KEY=supersecret
+
+# Google Maps API key for e.g. city autocomplete
+GOOGLE_MAPS_API_KEY=YourGoogleMapsAPIKey
```

Comment:
Shall we add the one you created for `localhost:3000` here? Otherwise it would override the setting in `config/google_maps.rb`.